Picture.py

```diff
@@ -33,10 +34,14 @@ def getTitle(self):
     def setTitle(self, name):
         self.title = name
 
-#    def getPixels(self):
-#        pixels = np.asarray(self.picture)
-#        # pixels = np.reshape(pixels, self.height, self.width)
-#        return pixels
+    def getPixels(self):
+        return self.image.getdata()
+
+    def getPixel(self, x, y): 
+        imageList = self.getPixels()
+        val = imageList[x+(y*self.width)]
+        pix = Pixel(self.image, x, y)
+        return pix
```

Comment:
Probably want to cast self.image.getdata() to a list and then create a list of Pixel objects from the list.